src/view/trip-filters-view.js

```diff
@@ -0,0 +1,37 @@
+import {createElement} from '../render.js';
+
+const filterName = ['everything', 'future', 'present', 'past'];
+const checkedName = ['checked', '', '', ''];
+const disabledName = ['', 'disabled', '', '',];
+
+let i = -1;
+
+function createTripFiltersTemplate() {
+  i++;
+  return (
+    `<div class="trip-filters__filter">
```

Comment:
<form class="trip-filters" action="#" method="get">
  
      <div class="trip-filters__filter">
      <input id="filter-everything" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="everything" checked="">
      <label class="trip-filters__filter-label" for="filter-everything">everything</label>
    </div>
      <div class="trip-filters__filter">
      <input id="filter-future" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="future">
      <label class="trip-filters__filter-label" for="filter-future">future</label>
    </div>
      <div class="trip-filters__filter">
      <input id="filter-present" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="present">
      <label class="trip-filters__filter-label" for="filter-present">present</label>
    </div>
      <div class="trip-filters__filter">
      <input id="filter-past" class="trip-filters__filter-input  visually-hidden" type="radio" name="trip-filter" value="past">
      <label class="trip-filters__filter-label" for="filter-past">past</label>
    </div>
  <button class="visually-hidden" type="submit">Accept filter</button>
</form>